README.md

```diff
@@ -1,5 +1,124 @@
 # Terraform OPA policies examples
 
 [![OPA tests](https://github.com/Scalr/sample-tf-opa-policies/workflows/OPA/badge.svg)](https://github.com/Scalr/sample-tf-opa-policies/actions?query=workflow%3AOPA)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/Scalr/sample-tf-opa-policies/pulls)
+[![Documentation Status](https://readthedocs.com/projects/scalr-athena/badge/?version=latest)](https://docs.scalr.com/en/latest/opa.html)
+
+A sample collection of OPA policies to test against Terraform run in Scalr.
+
+## Resources
+
+* Blog series for getting started with OPA for Terraform in Scalr : https://scalr.com//blog/opa-series-part-1-open-policy-agent-and-terraform/
```

Comment:
Extra slash here - `scalr.com//blog`.
(Is the article not published yet? The link goes to 404 atm)

Reply:
I also find it a bit weird to have a whitespace before colon throughout the document, but I'm not sure it is a rule in English punctuation

Reply:
Will fix / and :